package.json

```diff
@@ -9,8 +9,9 @@
     "url": "https://github.com/Zarel/Pokemon-Showdown-Client.git"
   },
   "scripts": {
-    "lint": "eslint --config=.eslintrc.js --cache --cache-file=eslint-cache/base js/ && eslint --config=build-tools/.eslintrc.js --cache --cache-file=eslint-cache/build build-tools/update build-tools/build-indexes && tslint --project .",
-    "test": "npm run lint && tsc && node build && mocha test/*.js",
+    "lint": "eslint --config=.eslintrc.js --cache --cache-file=eslint-cache/base js/ && eslint --config=build-tools/.eslintrc.js --cache --cache-file=eslint-cache/build build-tools/update build-tools/build-indexes && tslint --project src && tslint --project api",
+    "test": "npm run lint && tsc -p src && tsc -p api && node build && mocha test/*.js",
+    "fulltest": "npm test && node api/build && mocha .api-dist/test/*.js",
```

Comment:
This should also run the regular client tests, and linting, yeah?

Reply:
Also, GitHub Actions should be configured to run linting, typechecking for both src/ and api/, and tests for both the client and the loginserver. I'm not sure what your plan for `npm` scripts is.

Reply:
GitHub Actions cannot run most loginserver tests. And yes, fulltest runs both regular tests and linting.